docs/multiarch.md

```diff
@@ -0,0 +1,58 @@
+# How do I ship an operator that supports multiple node architectures?
+
+It is assumed that all operators run on linux hosts, but can be configured to manage workloads on other architecturs if worker nodes are available.
+
+Which os/arches are supported by the operator can be indicated by adding labels to the ClusterServiceVersion providing the operator.
+
+## Supported Labels
+
+Labels indicating supported os and arch are defined by:
+
+```yaml
+labels:
+    operatorframework.io//arch.<GOARCH>: supported
```

Comment:
Are we sure all of the possible values in GOARCH and GOOS are going to result in a valid label name?

Reply:
Hmm, what is currently in the list looks like just alphanumeric, is that a requirement so nothing potentially added in the future would bite us?

Reply:
I don't think go has committed to that, but if something changes in the future we can always update our doc. It's also possible we will want to support os/arches that golang doesn't, so we'll need a "real" list then.

Reply:
`supported` is the only value, correct?  I assume if `GOARCH` is `false`, it simply won't be listed?  

Somewhat curious why you prefer:

```yaml
labels:
    operatorframework.io/arch.<GOARCH>: supported
    operatorframework.io/os.<GOOS>: supported
#
labels:
    operatorframework.io/os.windows: supported
    operatorframework.io/os.linux: supported
```
over something like:

```yaml
labels:
    operatorframework.io/supported/arch.<GOARCH>: true 
    operatorframework.io/supported/os.<GOOS>: true
#
labels:
    operatorframework.io/supported/os.windows: true
    operatorframework.io/supported/os.linux: true 
```
Does the `string` value imply there may be other values in the future?

Reply:
I tend to default to making things enums to allow future expansion - but my assumption here is that most tooling would just check for the presence of the key.

If I try to predict the future now, I could imagine different values for whether an operator requires a particular arch for worker nodes, master nodes, or both?

Reply:
Ok great, yeah that makes sense.  I assume you would have some kind of `deprecation` for `supported`, should it transition to `worker, master, both` kinds of values (or a list).  Not sure if you would just want to start with `worker master` as a value now so that there isn't a need to deprecate, though it may be inaccurate to state `both` if there isn't a mechanism currently to verify both.